packages/app/package.json

```diff
@@ -10,6 +10,7 @@
     "@backstage/plugin-api-docs": "^0.2.0",
     "@backstage/plugin-catalog": "^0.2.0",
     "@backstage/plugin-circleci": "^0.2.0",
+    "@backstage/core-api": "^0.2.0",
```

Comment:
you should not depend directly on `core-api`- you should just depend on `core` which re-exports it